llvm-10.0.1/llvm-crash-analyzer/include/Analysis/TaintAnalysis.h

```diff
@@ -63,7 +63,8 @@ struct TaintInfo {
 
 class TaintAnalysis {
 private:
-  StringRef DotFileName = "";
+  StringRef TaintDotFileName = "";
```

Comment:
The implicit constructor for StringRef creates an empty stringref.   So, could we skip initialization ? 
Also, we could then use empty() API of StringRef to check if this is empty (instead of comparing with "".

Reply:
I agree, thanks for the comment! I think this PR needs to be approved again, since I added changes to address your comment.